app/javascript/app/document-capture/components/acuant-capture.jsx

```diff
@@ -39,7 +39,7 @@ function AcuantCapture({ label, hint, bannerText, value, onChange, className })
       )}
       <FileInput
         label={label}
-        hint={hint}
```

Comment:
Looks like we need to remove `hint` as a param to this function
```
/home/circleci/identity-idp/app/javascript/app/document-capture/components/acuant-capture.jsx
  12:33  error  'hint' is defined but never used  no-unused-vars
```

Reply:
> Looks like we need to remove `hint` as a param to this function
> 
> ```
> /home/circleci/identity-idp/app/javascript/app/document-capture/components/acuant-capture.jsx
>   12:33  error  'hint' is defined but never used  no-unused-vars
> ```

Whoops 😄  5d3aa3612